cts/cts-scheduler.in

```diff
@@ -1120,6 +1120,8 @@ TESTS = [
           "Make sure partial migrations are handled before ops on the remote node" ],
         [ "remote-partial-migrate2",
           "Make sure partial migration target is prefered for remote connection" ],
+        [ "remote-partial-migrate3",
```

Comment:
realizing this doesn't qualify as a partial migration, which is really more of an "in-progress migration"